### ISSUE TITLE
Set speedp and slowdown track speed.

### ISSUE THREAD

**Issue by ghost** [Foohy/gm-rollercoaster#18]
It would make it easier if you can set how much you want your cart to speed up or slow down. Thanks. Fantastic mod btw.

**Comment by Foohy** [comment]
Yep, the functionality is actually there already I just need an interface for it, but I honestly don't know where I can put it without cluttering up the existing tool any more without stuffing it in some corner no one would look in.

**Comment by ghost** [comment]
You can make it if the person selects the speed up or break the box extends with the options. Or have it switch in and out with the launch option. I'm not good at taking a though and turning it into words. I attached an image on what I'm trying to describe.
![2014-07-09_00009](https://cloud.githubusercontent.com/assets/5721713/3534256/f59c3f2a-07e4-11e4-8d90-00719cceaca1.jpg)

**Comment by Foohy** [comment]
Yeah that might be the way to go, and perhaps in addition to that have the option to change it via right clicking on the node and through that context menu.

**Comment by ghost** [comment]
Also, when I hit the launch key it cause like all my mods to crash/create lua errors.
